src/commands/login.js

```diff
@@ -1,53 +1,46 @@
-let cachedUsername;
```

Comment:
I am not sure this variable was ever set, so I removed it and everything consuming it. I believe the `sessionStore` is an expansion of its intended behavior.